### PR TITLE
feat: allow multiple outputs for same input

### LINKED ISSUE #99: [Bug] Copy multiple time the same file does one input only
I tried to do something like this:
```js
plugins: [
  new CopyWebpackPlugin([
    {
      from: 'a.js',
      to: 'b.js',
    },
    {
      from: 'a.js',
      to: 'c.js',
    },
  ]),
],
```

When doing that, only one output file is created. 
I had to do this to make it work:
```js
plugins: [
  new CopyWebpackPlugin([
    {
      from: 'a.js',
      to: 'b.js',
    },
  ]),
  new CopyWebpackPlugin([
    {
      from: 'a.js',
      to: 'c.js',
    },
  ]),
],
```

### PR DESCRIPTION
**Pull request by jdeniau**
Fixes #99

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
@jdeniau Please rebase to latest `master`, so we can triage this further :)

**Comment by jdeniau** [comment]
@michael-ciniawsky done

File: src/writeFile.js [hunk 1]
```diff
@@ -65,13 +65,18 @@ export default function writeFile(globalRef, pattern, file) {
...
                 info(`skipping '${file.webpackTo}', because it hasn't changed`);
                 return;
             } else {
                 debug(`added ${hash} to written tracking for '${file.absoluteFrom}'`);
                 written[file.absoluteFrom] = {
-                    [hash]: true
+                    [file.webpackTo]: {
+                        [hash]: true
+                    }
                 };
```
Comment (d3viant0ne) [inline src/writeFile.js]: Do you have a specific use case that can't be accomplished using `copyUnmodified`.

Based on the test case below, you could do the same thing without adding new functionality.

Comment (jdeniau) [reply src/writeFile.js]: I improved the test to add a transform method to the plugin. This way it does not work even if you add `copyUnmodified` option.

For the record, when I had the bug, it was by copying the same file with the same transformation (basically babel + uglify) in two different locations for history purpose (two different files became only one, but both are referenced)

**Comment by jdeniau** [comment]
For the record the test is failing in Node 4 only with a gcc bug, I think you can restart it and it should work.

Thank

**Comment by d3viant0ne** [comment]
Triggered a rebuild on Travis.

**Comment by evilebottnawi** [comment]
Duplicate https://github.com/webpack-contrib/copy-webpack-plugin/pull/165

**Pull request closed without merging**